### DISCUSSION TITLE
Spinner for loading botton

### DISCUSSION THREAD

**Discussion by neerosity** [tailwindlabs/heroicons#603] [Ideas]
There is no icon for spinner or loading or progress.

**Comment by vhscom** [comment]
If you landed here because you're looking for a low-effort spinner and are using Tailwind CSS, try `cursor-wait` on your `disabled` submit buttons. You'll get a nice-looking spinner and don't have to worry about setting a fixed width on your buttons.

**Comment by WesleyKapow** [comment]
Bump, a loading spinner is a widely used icon that I'd expect all libraries to have 🙏

**Comment by ndri** [comment]
I added `arc-third` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=arc-third
